src/constants/Style.js

```diff
@@ -10,6 +10,10 @@ module.exports = {
     FONT: '#2E323F',
     LIGHT_FONT: '#999999',
 
+    BASE_ARC: '#F5F5F5',
+    DEFAULT_ARC: '#359BCF',
+    DEFAULT_DOT: '#F7B148',
```

Comment:
We don't need to create `DEFAULT_ARC` or `DEFAULT_DOT`. Those values are already available with `PRIMARY` and `SECONDARY`.